symplyphysics/core/operations/single_argument_operation.py

```diff
@@ -0,0 +1,21 @@
+from typing import Self, Any
+from sympy import Basic, Symbol as SymSymbol
+from ..symbols.symbols import DimensionSymbolNew, next_name
+from ..dimensions import collect_factor_and_dimension
+
+
+class SingleArgumentOperation(DimensionSymbolNew, SymSymbol):  # pylint: disable=too-many-ancestors
+    """
+    This class is for dimensional symbols that act as an operation with a
+    single argument. Its subclasses are intended to be used in custom
+    code printers. The argument is placed at position `0` of the `args`
+    property.
+    """
+
+    def __new__(cls, _argument: Basic, **assumptions: Any) -> Self:
+        return SymSymbol.__new__(cls, next_name("SYS"), **assumptions)
```

Comment:
Use `SYM`. I think `SYS` is for coordinate systems.

Reply:
You're right